score-http/score-http/src/main/java/org/oagi/score/gateway/http/api/bie_management/service/BieService.java

```diff
@@ -158,6 +189,8 @@ public BieCreateResponse createBie(AuthenticatedPrincipal user, BieCreateRequest
     public PageResponse<BieList> getBieList(AuthenticatedPrincipal user, BieListRequest request) {
         PageRequest pageRequest = request.getPageRequest();
         AppUser requester = sessionService.getAppUserByUsername(user);
+        List<ULong> userTenantIds = tenantService
```

Comment:
Admin user should be able to see any BIEs whether it has assigned the tenant roles or not. (https://oagiscore.atlassian.net/wiki/spaces/SCORE/pages/4416962592/Multi-tenant#Multi-tenant-Score-design)